CONTRIBUTING.md

```diff
@@ -1,297 +1,385 @@
 
-If the package needs to perform tasks that cannot be done with command line switches, e.g. clicking away a prompt during installation that cannot be suppressed like for example in the [dropbox](https://chocolatey.org/packages/dropbox) package, you can use [AutoHotkey](https://chocolatey.org/packages/autohotkey.portable). Make [autohotkey.portable](https://chocolatey.org/packages/autohotkey.portable) a dependency of the package. Prefer AutoHotkey over [AutoIt](https://chocolatey.org/packages/autoit.commandline), because AutoHotkey is more lightweight, FOS and more actively developed than AutoIt.
+#### Icons
 
-### 1.4.1 Work on all locales
+A package must have an icon available if the packaged software has an icon. This icon must be named the same as the package and is placed in the [icons](https://github.com/chocolatey-community/chocolatey-packages/tree/master/icons) directory.
+If the package name ends with either `.install` or `.portable`, the suffix may be ignored in the icon name.
+When an icon is added to this folder with the correct name, it will **automatically** be set in the metadata file and the Readme file when our build server updates the package, if the metadata file contains an `<IconUrl>` element.
 
-Script must work on every locale available for Windows, so be careful when using strings of text of window in the script that could be different in another language.
+**IMPORTANT: If no icon is available, the comment `<!-- IconUrl: Skip check -->` should be added to the metadata file**
 
-### 1.4.2 Avoid brittle scripts
+#### Chocolatey Compatibility
 
-Do not create brittle scripts that work only when user doesn't interfere. All script elements should be as precise as possible - for instance, instead of using [Send](https://autohotkey.com/docs/commands/Send.htm) function which will work correctly only if the desired window is active, use [ControlSend](https://autohotkey.com/docs/commands/ControlSend.htm) which doesn't require window activation or use [BlockInput](https://autohotkey.com/docs/commands/BlockInput.htm) for short period of time.
+All packages are expected to be compatible with the Chocolatey CLI version, available 12 months earlier than the last change of the package.
+That means if you change the metadata or one of the package scripts **on January 12th, 2023**, it should be compatible with Chocolatey CLI 0.11.3 and later released versions.
+If that is not possible, and the package needs a version of Chocolatey CLI released later, it will need to include a dependency for that Chocolatey CLI version.
 
-# 2. Source Files
+### Chocolatey Scripts
 
-### 2.1 Encoding
+All PowerShell scripts are expected to be compatible with PowerShell v2 and later (not PowerShell Core). If this is not possible, a dependency on the package `PowerShell` is required, with its minimum version set to the earliest version that the scripts in the package are expected to work with.
```

Comment:
```suggestion
All PowerShell scripts are expected to be compatible with Windows PowerShell v2 through v5.1. If this is not possible, a dependency on the package `PowerShell` is required, with its minimum version set to the earliest version that the scripts in the package are expected to work with.
```

Is it worthwhile mentioning `Windows` PowerShell and being explicit about the versions instead of saying 2+ and then saying not Core (which technically 7.x isn't...)?